cgi-fcgi/cgi-fcgi.c

```diff
@@ -760,7 +760,7 @@ int main(int argc, char **argv)
     }
 
     if(OS_LibInit(stdinFds)) {
-        fprintf(stderr, "Error initializing OS library: %lu\n", OS_Errno);
+        fprintf(stderr, "Error initializing OS library: %lu\n", (unsigned long)OS_Errno);
```

Comment:
Done the same way with commit 545f2a86, although ISO C standard says _errno_ is of type _int_.